examples/tensorflow-privacy/README.md

```diff
@@ -27,40 +29,35 @@ This will create a new directory called `tensorflow-privacy` containing the foll
 -- README.md
 ```
```

Comment:
this and the above lines are no longer needed (since it's presented below already in the correct/new format)